appveyor.yml

```diff
@@ -5,7 +5,7 @@ environment:
 
   global:
       PYTHON: "C:\\conda"
-      MINICONDA_VERSION: "3.5.5"
+      MINICONDA_VERSION: "latest"
```

Comment:
This seems likely to be problematic in the future, but as long as it works right now...